dev/integration_tests/spell_check/integration_test/integration_test.dart

```diff
@@ -197,5 +197,5 @@ void main() {
     // We expect it to be rare for the first request to complete before the
     // second, so no text should be saved as of now.
     expect(defaultSpellCheckService.lastSavedResults, null);
-  });
+  }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.android }));
```

Comment:
@jmagman Is there a way for me to limit this test to Android only? This test passes on Android but fails on iOS because the iOS spell checker seems to be significantly faster, and may not run into the case that this test concerns.